catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/impl/MockSource.java

```diff
@@ -109,7 +111,11 @@ public String getVersion() {
 
     @Override
     public SourceResponse query(QueryRequest query) {
-        return null;
+        return sourceResponse;
+    }
+
+    public void setQueryResponse(SourceResponse sourceResponse) {
```

Comment:
It's generally frowned upon to add methods for the purposes of testing, but if there are no alternatives, we should at least make it protected